connectors/connectors.sh

```diff
@@ -72,7 +84,7 @@ get_connector_url() {
   fi
 
   # bigquery
-  if [[ $(min_version "$DATAPROC_VERSION" 2.0) == 2.0 ]]; then
+  if [[ $(min_version "$DATAPROC_IMAGE_VERSION" 2.0) == 2.0 ]]; then
```

Comment:
Some older 1.5 and 2.0 images might not have DATAPROC_IMAGE_VERSION defined, instead they use DATAPROC_VERSION. So, for backward compatibility, you might need to check both.

Reply:
done